### PR TITLE
v0.9 release notes

### PR DESCRIPTION
**Pull request by kevinintel**

### CODE REVIEW

File: release_notes/v0.9.md [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+- Other examples
+    - Clean deprecated VisualQnA code([87617e7](https://github.com/opea-project/GenAIExamples/commit/87617e7))
+    - Using TGI official release docker image for intel cpu([b2771ad](https://github.com/opea-project/GenAIExamples/commit/b2771ad))
+    - Add VisualQnA UI([923cf69](https://github.com/opea-project/GenAIExamples/commit/923cf69))
+    - fix container name([5ac77f7](https://github.com/opea-project/GenAIExamples/commit/5ac77f7))
+    - Add VisualQnA docker for both Gaudi and Xeon using TGI serving([2390920](https://github.com/opea-project/GenAIExamples/commit/2390920))
+    - Remove LangSmith from Examples([88eeb0d](https://github.com/opea-project/GenAIExamples/commit/88eeb0d))
+    - Modify the language variable to match language highlight.([f08d411](https://github.com/opea-project/GenAIExamples/commit/f08d411))
+    - Remove deprecated folder.([7dd9952](https://github.com/opea-project/GenAIExamples/commit/7dd9952))
+
```
Comment (bharagha) [inline release_notes/v0.9.md:78]: Added ProductivitySuite ([947cbe3](https://github.com/opea-project/GenAIExamples/commit/947cbe))

File: release_notes/v0.9.md [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+## What’s New in OPEA v0.9
+
+- Broaden functionality
+    - Initialize two Agent examples
+    - Support the experimental Fine-tuning microservice
+    - Use Llama-Guard-2-8B as default Guardrails model and bge-large-zh-v1.5 as default mosec embedding & reranking model
+    - Provides telemetry functionalities for metrics and tracing using Prometheus, Grafana, and Jaeger
+    - Add a benchmark script to assess the performance of the examples
+
+- Enhance Deployment
```
Comment (mkbhanda) [inline release_notes/v0.9.md:11]: s/Enhance Deployments/Enhancements

File: release_notes/v0.9.md [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+- Broaden functionality
+    - Initialize two Agent examples
+    - Support the experimental Fine-tuning microservice
+    - Use Llama-Guard-2-8B as default Guardrails model and bge-large-zh-v1.5 as default mosec embedding & reranking model
+    - Provides telemetry functionalities for metrics and tracing using Prometheus, Grafana, and Jaeger
+    - Add a benchmark script to assess the performance of the examples
+
+- Enhance Deployment 
+    - Enable support for launching container images as a non-root user
+    - Support for authentication and authorization with GMC
```
[the remark this answers is not in the document]
Comment (mkbhanda) [reply release_notes/v0.9.md:13]: Providing some additional information for integration

- Documented use of Node Resource Interface (NRI), both balloon and topology aware, to realize improved performance. 
- Added Kubernetes Horizontal Pod Autoscaling support in the helm charts that leverages CPU metrics from OPEA backend services.
- GenAI Microservices Connector (GMC) now supports data preparation for ChatQnA, response streaming, resource cleanup, dynamic adjustment, and detailed status reports. GMC successfully tested on Nvidia GPUs.
- Ease of Use: Introduced Model Caches through the use of Persistent Volume Claims, readiness and liveness probes for Helm charts and manifests and proxy handling. iCharts have been refactored and many more CI tests added. We have now a Gaudi base operator for Kubernetes environments.
- Added benchmark framework and scripts for performance and scale tests for one/two/four node Gaudi clusters. Leverages Locust.

File: release_notes/v0.9.md [hunk 4]
```diff
@@ -0,0 +1,302 @@
+# OPEA Release Notes v0.9
+## What’s New in OPEA v0.9
+
+- Broaden functionality
+    - Initialize two Agent examples
+    - Support the experimental Fine-tuning microservice
+    - Use Llama-Guard-2-8B as default Guardrails model and bge-large-zh-v1.5 as default mosec embedding & reranking model
+    - Provides telemetry functionalities for metrics and tracing using Prometheus, Grafana, and Jaeger
+    - Add a benchmark script to assess the performance of the examples
+
```
Comment (mkbhanda) [inline release_notes/v0.9.md:10]: - Re-organized documentation towards making developer and user experience smoother. Introduced the use of Sphinx to build and publish docs.

Comment (mkbhanda) [reply release_notes/v0.9.md:10]: @dbkinder and @tomlenth please see if you would like to add more above. We keep highlights pretty brief.

@kevinintel would you kindly pull into the details section the activity in the doc repository.

**Review by mkbhanda — COMMENTED** [review]
Some edits suggested

File: release_notes/v0.9.md [hunk 5]
```diff
@@ -0,0 +1,302 @@
...
+- Enhance Deployment 
+    - Enable support for launching container images as a non-root user
+    - Support for authentication and authorization with GMC
+    - Add Kubernetes support for AudioQnA and VisualQnA example
+
```
[the remark this answers is not in the document]
Comment (kevinintel) [reply release_notes/v0.9.md:26]: done

File: release_notes/v0.9.md (hunk 4 above)
Comment (kevinintel) [reply release_notes/v0.9.md:10]: add githubio url, but the title of the repo looks not formal. David can think how to add it.

File: release_notes/v0.9.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (kevinintel) [reply release_notes/v0.9.md]: commit again, please review

File: release_notes/v0.9.md (hunk 2 above)
Comment (kevinintel) [reply release_notes/v0.9.md]: commit again, please review

**Review by hshen14 — APPROVED** [review]

**Review by lvliang-intel — APPROVED** [review]

**Pull request merged**